components/higher-order/with-dnd/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { DragDropContext, DragSource, DropTarget } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/* Testing... */
+const ModifiedBackend = ( ...args ) => {
```

Comment:
Experimenting with a custom provider for react-dnd due to the conflicts created with dropzones.